### PR TITLE
Add one more test case.

### PR DESCRIPTION
**Pull request by chriseth**

### CODE REVIEW

File: test/libsolidity/semanticTests/userDefinedValueType/calldata_to_storage.sol [hunk 1]
```diff
@@ -25,19 +25,19 @@ contract C {
 // ----
 // s() -> 0, 0, 0x00, 0
 // f((uint8,uint16,bytes2,uint8)): 1, 0xff, "ab", 15 ->
-// gas irOptimized: 110778
+// gas irOptimized: 44860
 // gas legacy: 112851
-// gas legacyOptimized: 110766
+// gas legacyOptimized: 44923
```
Comment (chriseth) [inline test/libsolidity/semanticTests/userDefinedValueType/calldata_to_storage.sol:30]: This sounds like the gas costs were computed without the storage slot change?? How did it succeed in the PR?

Comment (hrkrshnn) [reply test/libsolidity/semanticTests/userDefinedValueType/calldata_to_storage.sol:30]: Yeah, this looks too suspicious. The cost is almost half.

Comment (chriseth) [reply test/libsolidity/semanticTests/userDefinedValueType/calldata_to_storage.sol:30]: Ah no it's fine - the test was added to develop before we fixed anything.

Comment (hrkrshnn) [reply test/libsolidity/semanticTests/userDefinedValueType/calldata_to_storage.sol:30]: If that is the case, then I think the gas for legacy should also change.

**Pull request merged**